aries_vcx/src/protocols/issuance/issuer/state_machine.rs

```diff
@@ -405,7 +405,8 @@ impl IssuerSM {
                             }),
                         })
                     }
-                    // todo: dont' transition, throw error, add to_failed transition() api which SM consumer can call
+                    // todo: 1. Don't transition, throw error, add to_failed transition() api which SM consumer can call
+                    //       2. Also create separate "Failed" state
                     Err(err) => {
                         let problem_report = build_problem_report_msg(Some(err.to_string()), &self.thread_id);
                         error!(
```

Comment:
This error log is misleading, but more importantly, the problem reports are still not being sent in case an error occurred, and nodejs consumer has no easy way to obtain and send them.

Reply:
Restored problem report sending on `libvcx_core` and `agent` level

Reply:
Good 👍